src/lock/win32/lock.c

```diff
@@ -5,7 +5,9 @@
  */
 #undef  _WIN32_WINNT
 #define _WIN32_WINNT 0x0400
+#define WIN32_LEAN_AND_MEAN
 #include <windows.h>
+#undef WIN32_LEAN_AND_MEAN
```

Comment:
Actually I just kept it consistent with existing files (they had WIN32_LEAN_AND_MEAN defined locally). Now I moved define to project settings. Still I'm not sure how it would affect mingw build. Maybe some makefile should be changed as well?